canonical/ubuntu.com
# Create new Grundium case study

## Done

- Create new case study using macros
- Wrap default CTA in `cta_content` block, added a new macro for CTA
- Update quote macros with Vanilla's changes

## QA

- Go to https://ubuntu-com-14338.demos.haus/case-study/grundium-ubuntu-pro-for-devices
- Check that content matches [copy doc](https://docs.google.com/document/d/1wM1vo24SFowPaAnxWV9wfU-w6PVDq7RczM_5ZOb8OZM/edit#heading=h.eto59uxtlv76) and [design](https://www.figma.com/design/rEDsQqLe8SfnvoxCcXJYLr/Case-studies?node-id=1-1276&node-type=frame&t=ji6rTzF77DDB3pM2-0) 
- Go to https://ubuntu-com-14338.demos.haus/case-study/esa
- Check that the footer CTA is default "Contact Us" and the quotes are rendering as expected

## Issue / Card

Fixes [WD-14824](https://warthogs.atlassian.net/browse/WD-14824)

## Screenshots

[If relevant, please include a screenshot.]


## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)


[WD-14824]: https://warthogs.atlassian.net/browse/WD-14824?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
